student.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'person'
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, classroom = 'Unknown')
+    super(age, name)
+    @classroom = classroom
+  end
```

Comment:
- [ ] While initializing the student class, the constructor should extends that of the person class. the name and the parent_permission should be passed when creating a student. Kindly fix that.